### PR TITLE
Update project setup and port to Python 3

### LINKED ISSUE #226: Port mopidy-gmusic to Python 3
Tracking issue for the porting of this extension to Python 3.

### PR DESCRIPTION
**Pull request by jodal**
Fixes #226

### CODE REVIEW

**Comment by belak** [comment]
Really wish this didn't change pretty much every quote in the project - makes it harder to tell what changes are actually related to Python 3... in any sense, I'll aim to get through this today.

Thanks for submitting this!

**Comment by jodal** [comment]
The reformatting with Black is a separate commit, so you can view the changes commit by commit to get a feeling for the changes.

File: mopidy_gmusic/commands.py [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
+        print(
+            "Go to the following URL to get an initial auth code, then "
+            + "provide it below: "
+            + flow.step1_get_authorize_url()
+        )
         print()
         try:
-            initial_code = raw_input('code: ')
+            initial_code = raw_input("code: ")
         except NameError:
```
Comment (belak) [inline mopidy_gmusic/commands.py]: If we're moving to only supporting python3, we shouldn't need to fall back here.

**Comment by belak** [comment]
Just the small comment, otherwise it looks good. Looks like the actual port was fairly painless.

File: mopidy_gmusic/commands.py (hunk 1 above)
Comment (jodal) [reply mopidy_gmusic/commands.py]: Good catch. Fixed now!

**Pull request merged**